src/NavBar.js

```diff
@@ -488,6 +493,25 @@ class NavBar extends React.Component {
       this.props.renderTitle;
     const navigationBarBackgroundImage = this.props.navigationBarBackgroundImage ||
       state.navigationBarBackgroundImage;
+
+    // by default, we want the background image to stretch over the whole navbar
+    // Note that if you change the overall navbar height, you'll have to change the background height too
+    const navigationBarBackgroundImageStyle = [
+      { ...Platform.select({
+          ios: {
+            height: IOS_DEFAULT_HEIGHT,
+          },
+          android: {
+            height: ANDROID_DEFAULT_HEIGHT,
+          },
+        }),
+        width: Dimensions.get('window').width
```

Comment:
You're absolutely right. Let me close this PR for now and come back with a better version that eliminates this dependency and covers all the scenarios.

Thanks for the reminder to never do late night PRs again. :-)